index.html

```diff
@@ -9,7 +9,73 @@
 	<link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
+	<nav class="navbar navbar-default navbar-inverse navbar-fixed-top">
+	  <div class="container-fluid">
+	    <!-- Brand and toggle get grouped for better mobile display -->
+	    <div class="navbar-header">
+	      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+	        <span class="sr-only">Toggle navigation</span>
+	        <span class="icon-bar"></span>
+	        <span class="icon-bar"></span>
+	        <span class="icon-bar"></span>
+	      </button>
+	      <a class="navbar-brand" href="#"><img src="img/name-01.png" alt="toplogo" class="header-logo"></a>
+	    </div>
 
+	    <!-- Collect the nav links, forms, and other content for toggling -->
+	    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+	      <ul class="nav navbar-nav navbar-right">
+	        <li class="active"><a href="#">Home</a></li>
+					<li><a href="#">Our Story</a></li>
+					<li><a href="#">Review</a></li>
+					<li><a href="#">Menu</a></li>
+					<li><a href="#">Contact Us</a></li>
+				</ul>
+	    </div><!-- /.navbar-collapse -->
+	  </div><!-- /.container-fluid -->
+	</nav>
+	<!-- Carousel begins -->
+	<div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+	  <ol class="carousel-indicators">
+	    <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+	    <li data-target="#carousel-example-generic" data-slide-to="1"></li>
+	    <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+	  </ol>
+
+	  <!-- Wrapper for slides -->
+	  <div class="carousel-inner" role="listbox"> <!-- wrapper for all slides, make sure all slides are within it -->
+	    <div class="item active" id="wood-bg">
+				<div class="flex-wrapper">
+	      	<img src="img/logo-01.png" alt="Cutlery logo" class="img-responsive center-block" id="carousel-logo">
+			  </div>
+			</div>
+	    <div class="item" id="seating-bg">
+	     <div class="carousel-caption">
+	        <h2>Eat Local.</h2>
+	        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugiat eum laudantium aut repellendus quos unde debitis, vero sunt! Quidem esse, dolores provident voluptates quod rerum ad omnis.</p>
+					<button class="btn btn-lg btn-warning">Learn More</button>
```

Comment:
I think it may be may computer but this button tag may be tabbed over incorrectly, along with the 2 divs below, and i think it caused a shift in the formatting below also. But besides that, everything looks good mang!